examples/user_interfaces/navigation.py

```diff
@@ -56,7 +56,7 @@ def trigger(self, *args, **kwargs):
 fig.canvas.manager.navigation.add_tool('List', ListTools)
 if matplotlib.rcParams['backend'] == 'GTK3Cairo':
     fig.canvas.manager.navigation.add_tool('copy', CopyToolGTK3)
-
+fig.canvas.manager.toolbar.add_tool('zoom', 'foo')
```

Comment:
I think it would be better, if we add `List` instead of `zoom`, it might be confusing to see two times the same tool